include/sway/tree/view.h

```diff
@@ -113,4 +113,8 @@ void view_close(struct sway_view *view);
 
 void view_update_outputs(struct sway_view *view, const struct wlr_box *before);
 
+void view_damage_whole(struct sway_view *view);
+
+void view_damage_from(struct sway_view *view);
```

Comment:
view damage from what?

Reply:
I don't know how to name this one. In rootston it's `view_damage` but I don't like this name because it doesn't imply the damage is taken from the _view damage_ (the function doesn't damage the whole view). I'd like to call it something like `desktop_damage_from_view` but it isn't consistent with other names.